code_samples/README.md

```diff
@@ -0,0 +1,37 @@
+# Code Samples
+
+This document explains Hazelcast Node.js Client code samples which you can use to learn how to use Hazelcast features.
```

Comment:
This folder (`/code_samples`) contains an extensive collection of Hazelcast Node.js Client code samples, which helps you to learn how to use Hazelcast features. The following lists the samples with their brief descriptions.